CHANGELOG.md

```diff
@@ -4,6 +4,107 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [v0.9.0] - 2022-05-13
+## ❗ BREAKING ❗
+
+### Remove the agent endpoint configuration for Zipkin [PR #1025](https://github.com/apollographql/router/pull/1025)
+Zipkin only supports the collector endpoint URL configuration.
+
+The Zipkin configuration changes from:
+
+```yaml
+telemetry:
+  tracing:
+    trace_config:
+      service_name: router
+    zipkin:
+      collector:
+        endpoint: default
+```
+
+to:
+
+```yaml
+telemetry:
+  tracing:
+    trace_config:
+      service_name: router
+    zipkin:
+      endpoint: default
+```
+
+### CSRF Protection is enabled by default [PR #1006](https://github.com/apollographql/router/pull/1006)
+A [Cross-Site Request Forgery protection plugin](https://developer.mozilla.org/en-US/docs/Glossary/CSRF) is enabled by default.
+
+This means [simple requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests) will be rejected from now on (they represent a security risk).
+
+The plugin can be customized as explained in the [CORS and CSRF example](https://github.com/apollographql/router/tree/main/examples/cors-and-csrf/custom-headers.router.yaml)
+
+### CORS default behavior update [PR #1006](https://github.com/apollographql/router/pull/1006)
+The CORS allow_headers default behavior changes from:
+  - allow only `Content-Type`, `apollographql-client-name` and `apollographql-client-version`
+to:
+  - mirror the received `access-control-request-headers`
+
+This change loosens the CORS related headers restrictions, so it shouldn't have any impact on your setup.
+
+## 🚀 Features ( :rocket: )
```

Comment:
```suggestion
## 🚀 Features ( :rocket: )
```

These emojis seem duplicative and look kind of odd when rendered.   Is this from our templates?